src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsTests/CustomNode.tsx

```diff
@@ -0,0 +1,49 @@
+import {Handle, Position} from 'reactflow';
+
+import {ClockCircleOutlined} from '@ant-design/icons';
+import {Tooltip} from 'antd';
+
+import {ExecutorIcon} from '@atoms';
+
+import {Text} from '@custom-antd';
+
+import {LocalStep} from '@models/testSuite';
+
+import {DotsDropdown} from '@molecules';
+
+import {TestNodeContainer} from './SettingsTests.styled';
+
+type CustomNodeProps = {
+  data: LocalStep & {
+    group: number | string;
+    removeNode: (id: string, col: number | string) => void;
```

Comment:
Is there a way to provide `removeNode` as separate prop?

Reply:
As far as I understand everything custom I want to provide to component should come inside `data` prop